### PR TITLE
Rus translation collection

### PR DESCRIPTION
**Pull request by mamedzuka**
## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes #[issue-number]
* Requires DSpace/DSpace#[pr-number] (if a REST API PR is required to test this)

## Description
Short summary of changes (1-2 sentences).

## Instructions for Reviewers
Please add a more detailed description of the changes made by your PR. At a minimum, providing a bulleted list of changes in your PR is helpful to reviewers.

List of changes in this PR:
* First, ...
* Second, ...

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [ ] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [ ] My PR passes [TSLint](https://palantir.github.io/tslint/) validation using `yarn run lint`
- [ ] My PR doesn't introduce circular dependencies
- [ ] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [ ] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new, third-party dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.

### CODE REVIEW

File: src/assets/i18n/ru.json5 [hunk 1]
```diff
@@ -2000,300 +1986,258 @@
...
-  "collection.edit.return": "Back",
-  
+  "collection.edit.return": "Назад",
   
   
+  // Что за Куратор?
   // "collection.edit.tabs.curate.head": "Curate",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.curate.head": "Curate",
+  "collection.edit.tabs.curate.head": "Куратор",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]: Это глагол же
```suggestion
  "collection.edit.tabs.curate.head": "Курировать",
```

File: src/assets/i18n/ru.json5 [hunk 2]
```diff
@@ -2000,300 +1986,258 @@
...
   // "collection.edit.tabs.roles.title": "Collection Edit - Roles",
   // TODO New key - Add a translation
-  "collection.edit.tabs.roles.title": "Collection Edit - Roles",
+  "collection.edit.tabs.roles.title": "Редактирование коллекции - Роли",
   
   // "collection.edit.tabs.source.external": "This collection harvests its content from an external source",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.external": "This collection harvests its content from an external source",
+  // Тут не уверен ВКЛЮЧАЕТ ли или СОБИРАЕТ контент
+  "collection.edit.tabs.source.external": "Данная коллекция включает в себя контент из внешних источников",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5:2050]: получает, а не включает

File: src/assets/i18n/ru.json5 [hunk 3]
```diff
@@ -2000,300 +1986,258 @@
...
   // "collection.edit.tabs.source.form.adminEmail": "Admin's address",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.adminEmail": "Admin's address",
+  "collection.edit.tabs.source.form.adminEmail": "Адрес админимтратора",
   
   // "collection.edit.tabs.source.form.errors.oaiSource.required": "You must provide a set id of the target collection.",
+  // Тут не понимаю, что есть set id
   // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.errors.oaiSource.required": "You must provide a set id of the target collection.",
+  "collection.edit.tabs.source.form.errors.oaiSource.required": "Вы должны указать set id набора выбранной коллекции.",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.edit.tabs.source.form.errors.oaiSource.required": "Требуется указать идентификатор набора (set id) выбранной коллекции.",
```

File: src/assets/i18n/ru.json5 [hunk 4]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.edit.tabs.source.form.forceSynchronization": "Force Synchronization",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.forceSynchronization": "Force Synchronization",
+  "collection.edit.tabs.source.form.forceSynchronization": "Принудительная Синхронизация",
   
   // "collection.edit.tabs.source.form.recordValidationEnabled": "Record validation",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.form.recordValidationEnabled": "Record validation",
+  "collection.edit.tabs.source.form.recordValidationEnabled": "Проверка записи",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]: может оставить «валидация»?

File: src/assets/i18n/ru.json5 [hunk 5]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.edit.tabs.source.form.ccAddresses": "CC's addresses",
+  // Что такое CC?
   // TODO New key - Add a translation
   "collection.edit.tabs.source.form.ccAddresses": "CC's addresses",
   
   // "collection.edit.tabs.source.head": "Content Source",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.head": "Content Source",
+  "collection.edit.tabs.source.head": "Источник Контента",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.edit.tabs.source.head": "Источник контента",
```

File: src/assets/i18n/ru.json5 [hunk 6]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.edit.tabs.source.notifications.discarded.content": "Your changes were discarded. To reinstate your changes click the 'Undo' button",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.notifications.discarded.content": "Your changes were discarded. To reinstate your changes click the 'Undo' button",
+  "collection.edit.tabs.source.notifications.discarded.content": "Вносимые изменения были отклонены. Для их восстановления нажмите кнопку 'Отменить'",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]: Низкопоклонство перед английскими кавычками!

```suggestion
  "collection.edit.tabs.source.notifications.discarded.content": "Вносимые изменения были отклонены. Для их восстановления нажмите кнопку «Отменить»",
```

File: src/assets/i18n/ru.json5 [hunk 7]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.edit.tabs.source.notifications.discarded.title": "Changed discarded",
+  // Тут нужна умная голова, я не понимаю что за одновременное изменение отклонено
   // TODO New key - Add a translation
   "collection.edit.tabs.source.notifications.discarded.title": "Changed discarded",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.edit.tabs.source.notifications.discarded.title": "Изменения отменены",
```

File: src/assets/i18n/ru.json5 [hunk 8]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.edit.tabs.source.notifications.invalid.content": "Your changes were not saved. Please make sure all fields are valid before you save.",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.notifications.invalid.content": "Your changes were not saved. Please make sure all fields are valid before you save.",
+  "collection.edit.tabs.source.notifications.invalid.content": "Вносимые изменения не были сохранены. Перед сохранением удостоверьтесь в правильности заполнения полей.",
   
   // "collection.edit.tabs.source.notifications.invalid.title": "Metadata invalid",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.notifications.invalid.title": "Metadata invalid",
+  "collection.edit.tabs.source.notifications.invalid.title": "Недействительные метаданные",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.edit.tabs.source.notifications.invalid.title": "Неправильные метаданные",
```

File: src/assets/i18n/ru.json5 [hunk 9]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.edit.tabs.source.notifications.saved.content": "Your changes to this collection's content source were saved.",
-  // TODO New key - Add a translation
-  "collection.edit.tabs.source.notifications.saved.content": "Your changes to this collection's content source were saved.",
+  "collection.edit.tabs.source.notifications.saved.content": "Вносимые в источник контента данной коллекции изменения были сохранены.",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.edit.tabs.source.notifications.saved.content": "Изменения в источник контента коллекции были сохранены.",
```

File: src/assets/i18n/ru.json5 [hunk 10]
```diff
@@ -2000,300 +1986,258 @@
...
   // "collection.form.title": "Name",
-  // TODO New key - Add a translation
-  "collection.form.title": "Name",
+  "collection.form.title": "Название",
   
   // "collection.form.entityType": "Entity Type",
+  // Юр. Лицо???
   // TODO New key - Add a translation
-  "collection.form.entityType": "Entity Type",
+  "collection.form.entityType": "Тип юр. лица",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]: Ой бля
```suggestion
  "collection.form.entityType": "Тип сущности",
```

File: src/assets/i18n/ru.json5 [hunk 11]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.form.errors.entityType.required": "Please choose an entity type for this collection",
+  // ТИП Юр. Лица???
   // TODO New key - Add a translation
-  "collection.form.errors.entityType.required": "Please choose an entity type for this collection",
+  "collection.form.errors.entityType.required": "Выберите тип юридического лица для данной коллекции",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.form.errors.entityType.required": "Выберите тип сущности для коллекции",
```

File: src/assets/i18n/ru.json5 [hunk 12]
```diff
@@ -2000,300 +1986,258 @@
...
   // "collection.form.provenance": "Provenance",
+  // Происхождение? Всмысле???
   // TODO New key - Add a translation
   "collection.form.provenance": "Provenance",
   
   // "collection.form.rights": "Copyright text (HTML)",
-  // TODO New key - Add a translation
-  "collection.form.rights": "Copyright text (HTML)",
+  // Копирайт разве переводится?
+  "collection.form.rights": "Авторские права (HTML)",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.form.rights": "Текст авторских права (HTML)",
```

File: src/assets/i18n/ru.json5 [hunk 13]
```diff
@@ -2000,300 +1986,258 @@
...
   
   // "collection.form.abstract": "Short Description",
-  // TODO New key - Add a translation
-  "collection.form.abstract": "Short Description",
+  "collection.form.abstract": "Краткое описание",
   
   // "collection.form.description": "Introductory text (HTML)",
-  // TODO New key - Add a translation
-  "collection.form.description": "Introductory text (HTML)",
+  "collection.form.description": "Введение (HTML)",
```
Comment (apalkoff) [inline src/assets/i18n/ru.json5]:
```suggestion
  "collection.form.description": "Вводный текст (HTML)",
```

**Review by apalkoff — CHANGES_REQUESTED** [review]
требуются доработки

File: src/assets/i18n/ru.json5 (hunk 4 above)
Comment (mamedzuka) [reply src/assets/i18n/ru.json5]: ок

File: src/assets/i18n/ru.json5 (hunk 6 above)
Comment (mamedzuka) [reply src/assets/i18n/ru.json5]: нормальные кавычки 
а треугольнички русские - бяка

**Review by apalkoff — COMMENTED** [review]
ку ку

**Comment by mamedzuka** [comment]
нене
чиллим
ща завал разгребу и эту тему доперевожу
и ещё что-нибудь 
пока PR пусть висит
я напишу, когда мерджить мона

**Comment by mamedzuka** [comment]
Можно проверять

**Review by apalkoff — APPROVED** [review]

**Pull request merged**